migration/src/m20240219_115532_add_extensions_column.rs

```diff
@@ -0,0 +1,59 @@
+use sea_orm_migration::{
+    prelude::*,
+    sea_orm::{ConnectionTrait, DatabaseBackend, Statement},
+};
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        let connection = manager.get_connection();
+
+        connection
+            .execute(Statement::from_string(
+                DatabaseBackend::Postgres,
+                "ALTER TABLE asset ADD COLUMN mint_extensions jsonb;".to_string(),
```

Comment:
Nit: is it preferable to use the SeaORM stuff to add things, rather than raw SQL?  That's how most of the other straightforward migrations code looks when adding columns.